dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationManager.java

```diff
@@ -260,6 +264,33 @@ public String getExpandLocation() {
         return this.expandLocation;
     }
 
+//    private void setAnnotationScanLibrary(String annotationScanLibraryValue) {
+//
+//        annotationScanLibrary.clear();
+//        String[] split = annotationScanLibraryValue.split(",");
+//
+//        for (String s : split) {
+//            if (s.trim().equals("ALL")) {
+//                annotationScanLibrary.clear();
+//                annotationScanLibrary.add(AnnotationScanLibraryValues.ALL);
+//                break;
+//            } else if (s.trim().equals("earLib")) {
+//                annotationScanLibrary.add(AnnotationScanLibraryValues.earLib);
+//            } else if (s.trim().equals("earLib")) {
+//                annotationScanLibrary.add(AnnotationScanLibraryValues.ManifestLib);
+//            }
+//        }
+//    }
```

Comment:
Why adding commented out code to the file?

Reply:
Tom and I made progress on enabling this for shared libraries, then hit problems and decided to focus on the easier stuff that would satisfy the customer first. We kept this so we don't loose the progress towards shared libraries.

Reply:
Should it move then to the other class since annotationScanLibrary doesn't exists in this class any longer?

Reply:
You know, I think I was wrong last time and this should be removed. We are preserving stuff but looking again. This bit is both basic and obsolete